dtfs-central-api/src/services/repositories/utilisation-reports-repo.js

```diff
@@ -2,16 +2,20 @@ const sortBy = require('lodash/sortBy');
 const db = require('../../drivers/db-client');
 const { DB_COLLECTIONS } = require('../../constants/dbCollections');
 
+/**
+ * @typedef {folder: string, filename: string, fullPath: string, url: string, contentType: string} AzureFileInfo
```

Comment:
I noticed in some tests our mock data doesn't have contentType and we don't care much about it, should we put it as an optional here? I'm not familiar with JSDoc types but assuming that `string | null` or `contentType?` is something we would do

Reply:
I've added it to tests now, as it's needed for downloading the file I don't think it should be nullable :)